src/pages/concept-registration-page/form-concept/form-concept-pure.component.tsx

```diff
@@ -52,10 +52,12 @@ export const FormConceptPure: React.FC<Props> = ({ concept, isValid }) => {
 
   return (
     <Form>
-      <LanguagePicker
-        languages={state.languages}
-        toggleInputLanguage={language => dispatch(toggleInputLanguage(language))}
-      />
+      <Can I="edit" of={{ __type: 'Language', publisher: publisherId }}>
```

Comment:
kan man ikk se alle språk med leserettighet??

Reply:
Ja, du kan se alle språk som er registrert. Men skjuler knappene i toppen av skjema for å velge språk, det er bare for de som har skriverolle som skal se.